packages/blade/src/components/Carousel/_decisions/decisions.md

```diff
@@ -0,0 +1,150 @@
+
+### CarouselItem
+
+- N/A - CarouselItem will just be a wrapper and have internal styling
+
+## Usage Example
+
+```jsx
+const Testimonial = () => {
+  return (
+    <Box display="flex" gap="spacing.2">
+      <Box>
+        <Title>Acquire Customers From New Customer Segments</Title>
+        <Text weight="bold">Subham Kumar</Text>
+      </Box>
+      <Img src="./avatar-1.png" />
+    </Box>
+  )
+};
+
+<Carousel
+  autoPlay
+  visibleItems={2}
+  navigationButtonPosition="side"
+  bleed="right"
+>
+  <CarouselItem>
+    <Testimonial />
+  </CarouselItem>
+
+  <CarouselItem>
+    <Testimonial />
+  </CarouselItem>
+
+  <CarouselItem>
+    <Testimonial />
+  </CarouselItem>
+</Carousel>
+```
+
+<img src="./example-usage-1.png" width="70%" />
+
+### Usage with Card
+
+
+```jsx
+const TestimonialCard = () => {
+  return (
+   <Card elevation="highRaised" padding="spacing.0">
+      <CardBody>
+        <Box display="flex" flexDirection="row">
+          <img
+            width="300"
+            height="auto"
+          />
+          <Box padding="spacing.7" display="flex" flexDirection="column">
+            <Heading size="large">Subham Kumar</Heading>
+            <Text marginTop="spacing.5">
+            Acquire Customers From New Customer Segments
+            </Text>
+            <Link href="#">Learn More</Link>
+          </Box>
+        </Box>
+      </CardBody>
+    </Card>
+  )
+};
+
+<Carousel
+  autoPlay
+  visibleItems={2}
+  navigationButtonPosition="side"
+  bleed="right"
+>
+  <CarouselItem>
+    <TestimonialCard />
+  </CarouselItem>
+
+  <CarouselItem>
+    <TestimonialCard />
+  </CarouselItem>
+
+  <CarouselItem>
+    <TestimonialCard />
+  </CarouselItem>
+</Carousel>
+```
+
+## Accessibility
+
+We need to follow the APG Pattern For [Carousel](https://www.w3.org/WAI/ARIA/apg/patterns/carousel/examples/carousel-2-tablist/)
+
+## Motion
+
+https://github.com/razorpay/blade/assets/35374649/6876f348-1b0a-4e9a-800a-300025f54b5c
+
+## Open Questions
+
+- Should we call it `navigationButtonStyle` or `navigationButtonVariant`? 
+
+## Libraries
+
+**web**
+
+On mweb & dweb we can use [CSS scroll snap](https://ishadeed.com/article/css-scroll-snap/) feature, this covers all our common usecase with bleed smooth scrolling etc.
+
+**react-native**
+
+on rnweb, we cannot use CSS scroll snap so might might need to opt for a library.
+But further exploration lead me to this [SO question](https://stackoverflow.com/questions/39849648/horizontal-scrollview-snapping-react-native) and [this expo example](https://snack.expo.io/H1CnjIeDb), turns our that ScrollView does support scroll snapping so if this works for our usecase it will be helpful.
+
+But we might need to see if this might cause an performance issue since we won't be using FlatList.
```

Comment:
Why can't we use FlatList? It has horizontal scroll

Reply:
FlatList doesn't support `snapToInterval, snapToAlignment`

Two things: 
- Even if snapToAlignment was supported in FlatList - we might again need to do hacks to convert the JSX to FlatList accepted data array. 
- Given that carousels should not have hundreds of elements, I wonder if FlatList is even needed or not. At most it might have 10 (even that sounds way too many)